recipes/nanort/all/conanfile.py

```diff
@@ -0,0 +1,30 @@
+from conan import ConanFile
+from conan.tools.files import copy, get
+
+import os
+
+
+class NanoRTConan(ConanFile):
+    name = "nanort"
+    description = "Single header only modern ray tracing kernel"
+    topics = ("graphics", "raytracing")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/lighttransport/nanort"
+    license = "MIT"
+    #version = "0bb8ab5"
+    # No settings/options are necessary, this is header only
+    exports_sources = "include/*"
+    # We can avoid copying the sources to the build folder in the cache
+    no_copy_source = True
+
```

Comment:
Please, add the layout and the package_id methods to the recipe:

```suggestion
    def layout(self):
        basic_layout(self, src_folder="src")

    def package_id(self):
        self.info.clear()
```

do not forget to import `from conan.tools.layout import basic_layout`